File: B/Python/RAM.py
class RAM():
    __capacity = 0
    __priceRAM = 0
    
    def __init__(self, capacity, price):
        self.__capacity = capacity
        self.__priceRAM = price
    
    def setCapacity(self, f):
        self.__capacity = f
    
    def getCapacity(self):
        return self.__capacity
    
    def setPriceRam(self, m):
        self.__priceRAM = m
    
    def getPriceRAM(self):
        return self.__priceRAM
    
    def printRAM(self):
        print("RAM           : "+str(self.__capacity))

File: B/PHP/RAM.php
<?php

class RAM{
    private $capacity = 0;
    private $priceRAM = 0;

    public function __construct($capacity = 0, $priceRAM = 0){
        $this->capacity = $capacity;
        $this->priceRAM = $priceRAM;
    }

    public function setCapacity($capacity){
        $this->capacity = $capacity;
    }

    public function getCapacity(){
        return $this->capacity;
    }

    public function setPriceRAM($priceRAM){
        $this->priceRAM = $priceRAM;
    }

    public function getPriceRAM(){
        return $this->priceRAM;
    }

    public function printRAM(){
        echo "RAM          : ". $this->capacity . "<br>";
    }
}

?>